express-get/server.ts

```diff
@@ -0,0 +1,54 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+    select *
+    from "films"
+    order by "replacementCost"
+    limit 2;
+  `;
+    const result = await db.query(sql);
+    const films = result.rows;
+    res.json(films);
+  } catch (err) {
+    next(err);
+  }
+});
+
+app.get('/api/films/:filmId', async (req, res, next) => {
+  try {
+    const sql = `
+    select *
+    from "films"
+    where "filmId" = $1
+    `;
+    const params = [req.params.filmId];
```

Comment:
It is often convenient to grab the params with destructuring:
`const { filmId } = req.params;`